docs/resources/index.rst

```diff
@@ -6,6 +6,7 @@ Resources
     :maxdepth: 3
     :hidden:
 
+    upgrading
```

Comment:
Couldn't think of a better place for this, but Resources is maybe a bit too hidden?